primitives/runtime-interface/proc-macro/src/lib.rs

```diff
@@ -63,6 +64,19 @@ impl Parse for Options {
 			} else if lookahead.peek(runtime_interface::keywords::no_tracing) {
 				let _ = input.parse::<runtime_interface::keywords::no_tracing>();
 				res.tracing = false;
+			} else if lookahead.peek(runtime_interface::keywords::feature_force_version) {
+				let _ = input.parse::<runtime_interface::keywords::feature_force_version>();
+				let _ = input.parse::<Token![=]>();
+				let feature_name = input.parse::<syn::Ident>()?;
+				let _ = input.parse::<Token![,]>();
+				let fonc_name = input.parse::<syn::Ident>()?;
+				let _ = input.parse::<Token![,]>();
+				let fonc_version = match input.parse::<syn::ExprLit>()? {
+					syn::ExprLit { lit: syn::Lit::Int(lit), .. } => lit.base10_parse::<u32>()?,
+					_ => return Err(lookahead.error()),
```

Comment:
I don't think this will result in a correct error, instead we should return an error saying we expect a `ExprLit`.